### ISSUE TITLE
With disable_sanitize_html=true <script> is no longer executed since v7.1

### ISSUE THREAD

**Issue by adeverteuil** [grafana/grafana#26535]
**What happened**:

Since upgrading to v7.1, `<script>` tags in Text panels are no longer executed even with `disable_sanitize_html=true`.
https://grafana.com/docs/grafana/latest/administration/configuration/#disable_sanitize_html

We can see the `<script>` tag is rendered in the DOM but it is not executed.

![screenshot 2020-07-21 1649](https://user-images.githubusercontent.com/3270755/88227639-e2e4be80-cc3b-11ea-9168-775d0e6cb4a7.png)

**How to reproduce it (as minimally and precisely as possible)**:

This is a simple Text panel in HTML mode that contains the following element:

```
<script>console.log("HELLO");</script>
```

It works in v7.0.5 but fails to run in v7.1.0.

Panel JSON:

```
{
  "fieldConfig": {
    "defaults": {
      "custom": {}
    },
    "overrides": []
  },
  "gridPos": {
    "h": 9,
    "w": 12,
    "x": 0,
    "y": 0
  },
  "id": 2,
  "options": {
    "mode": "html",
    "content": "<script>console.log(\"HELLO\");</script>"
  },
  "pluginVersion": "7.1.0",
  "targets": [
    {
      "refId": "A",
      "scenarioId": "random_walk"
    }
  ],
  "timeFrom": null,
  "timeShift": null,
  "title": "Panel Title",
  "type": "text",
  "datasource": null
}
```

**Anything else we need to know?**:

**Environment**:
- Grafana version: v7.1.0
- Data source type & version:
- OS Grafana is installed on: Fedora
- User OS & Browser: Firefox 78
- Grafana plugins:
- Others:

**Comment by Clarity-89** [comment]
Works for me on the latest master, tried Chrome and Firefox, so maybe it has been fixed? 
![Screenshot 2020-07-23 at 10 12 38](https://user-images.githubusercontent.com/8878045/88260443-0c99f600-cccd-11ea-9e41-7c0f7ba7c99d.png)

Also the refresh dashboard button doesn't re-execute the code, but instead Ctrl + S needs to be used inside the content box, not sure if that's intended, but a bit confusing at first.

**Comment by Clarity-89** [comment]
Turns out this has been fixed in https://github.com/grafana/grafana/pull/26413 and will be released in 7.1.1.
